openedx/features/enterprise_support/tests/test_utils.py

```diff
@@ -36,6 +36,7 @@ def setUpTestData(cls):
         ('notfoundpage', 0),
     )
     @ddt.unpack
+    @override_waffle_flag(ENTERPRISE_HEADER_LINKS, True)
```

Comment:
this implies there a waffle flag we need to enable in stage/prod?

Reply:
Yup! I'll turn these on, seems a safer rollout/rollback strategy just in case anything is off.